### DISCUSSION TITLE
My contribution does not counting how to solve it

### DISCUSSION THREAD

**Discussion by AashiqRahaman** [community/community#110434] [Repositories]
### Select Topic Area

Question

### Body

using Desktop git also from vs code I pushed my code file to my existing git repository. it's published to the repo well but it does not count as a contribution (green dot ). when I drag and drop one file to this same repo it counts as a contribution. Also when newly created a repo then also it counts as a contribution. help to solve it my liked email is all okay I cross-checked it.

![Screenshot 2024-02-29 020004](https://github.com/community/community/assets/156484668/03ae1f4d-360f-4a14-84ea-31d31f0ed9bc)
![Screenshot 2024-02-29 020023](https://github.com/community/community/assets/156484668/a61d468e-30c7-46e6-b14a-4879de9aa772)

**Comment by DeandrePetersen** [comment]
Firstly, GitHub counts contributions based on commits made to certain branches, typically the default branch (e.g., main or master). Ensure that you're pushing your changes to the correct branch, especially if you're working with multiple branches in your repository. Also GitHub may have a delay in updating the contributions graph. It's possible that the contribution will appear after some time has passed.

**Reply by AashiqRahaman** [reply]
Thanks for replied... it works

**Comment by AashiqRahaman** [comment]
Still, I am unable to figure out what the problem was. I just renamed my global user name from git desktops -  git bash. might be after that it works 
"git config --global user.email johndoe@example.com"
but one thing is that before then also I can able to push code to my repo but it was not counted as a contribution, I don't know what the issue is, but now it's solved👍

**Reply by babayl0729** [reply]
We're in the same boat. I just actually noticed today that mine is not working as well.
